hs_core/tests/api/rest/test_resource_system_meta.py

```diff
@@ -0,0 +1,42 @@
+# import json
+
+from rest_framework import status
+
+from hs_core.hydroshare import resource
+from .base import HSRESTTestCase
+
+
+class TestResourceSystemMetadata(HSRESTTestCase):
+
+    def setUp(self):
+        super(TestResourceSystemMetadata, self).setUp()
+
+        self.rtype = 'GenericResource'
+        self.title = 'My Test resource'
+        res = resource.create_resource(self.rtype,
+                                       self.user,
+                                       self.title)
+        self.pid = res.short_id
+        self.resources_to_delete.append(self.pid)
+
+    def test_get_sysmeta(self):
+        # Get the resource system metadata
+        sysmeta_url = "/hsapi/resource/{res_id}/scimeta/elements/".format(res_id=self.pid)
+        response = self.client.get(sysmeta_url)
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        # content = json.loads(response.content)
+
+    def test_put_sysmeta(self):
+        sysmeta_url = "/hsapi/resource/{res_id}/scimeta/elements/".format(res_id=self.pid)
+        put_data = {
```

Comment:
Don't we need to provide data for all metadata element types to make sure the metadata update works for all?